lib/action_mailer_matchers.rb

```diff
@@ -17,7 +17,7 @@ module ActionMailerMatchers
 
     def matching_mail_for(email, body, email_subject)
       deliveries.detect do |mail|
-        has_expected_email = mail.to.include?(email)
+        has_expected_email = mail.destinations.include?(email)
```

Comment:
What do you think about having more explicit expectations around this? I'm imagining something like this. Would this result in any false positives where we meant to send someone as the main `to` but instead sent it as `bcc`? 

```ruby
expect(user).to have_received_email(subject: "Subject")
expect(user).to have_received_cc_email(subject: "Subject")
expect(user).to have_received_bcc_email(subject: "Subject")
```

Reply:
The updates look great!

Reply:
Would you mind adding a small update to the usage section explaining that there are two additional methods? https://github.com/contently/action_mailer_matchers#usage

Reply:
Will do!